### PR TITLE
add arguments to main, 1st argument can be set as the input path.

### PR DESCRIPTION
**Pull request by dreyling**
left default, if no argument (path) is given.

### CODE REVIEW

**Review by TCThenna — CHANGES_REQUESTED** [review]
Add also cmd line input for "runmode", which (if given) overwrites the runmode from the config file

**Comment by dreyling** [comment]
Okay, now you should see the changes on run mode.. same request.

**Review by TCThenna — COMMENTED** [review]
I will add if() to only provide runmode but not input dir

File: src/main.cc [hunk 1]
```diff
@@ -1211,22 +1232,19 @@ int main(int argc, char *argv[0])
...
-  ifstream conf(s_config.c_str());
-  if(!conf)
-  {
-    std::cout << "../conf/config.txt file missing!" << std::endl;
-    return 1;
-  } else
-  {
-    conf >> runmode;
-  }
+  //std::string s_config = "../conf/config.txt";
```
Comment (TCThenna) [inline src/main.cc:1238]: instead of commenting out, next better just delete it. The old code is still git-ed, so we dont lose anything.

**Pull request merged**